### DISCUSSION TITLE
How to Upgrade SONOFF ZBDongle-E V2 from 7.3.1.0 build 176 (EZSP) to Firmware 7.4.x (ember)

### DISCUSSION THREAD

**Discussion by GWsmart** [Koenkk/zigbee2mqtt#23020] [Devices]
Hallo,
Home Assistant Core: 2024.5.5
Z2M Addon: 1.38.0
Coordinator-Typ: EZSP v12 (Sonoff ZBDongle-E)
Coordinator-Version: 7.3.1.0 build 176 
Zigbee Herdsman Konverter Version:    19.47.1
Zigbee Herdsman Version:     0.49.2

First of all, sorry for my bad english!

During the start of Z2M shows this massage:
`[2024-06-08 19:08:00] warning: 	zh:ezsp: Deprecated driver 'ezsp' currently in use, 'ember' will become the officially supported EmberZNet driver in next release. If using Zigbee2MQTT see https://github.com/Koenkk/zigbee2mqtt/discussions/21462`

So i want to migrade to ember, but i am not shure how to do!
First, is it necessary to do this?
Which Firmware i need? 7.4.1 or 7.4.2 and 115200.gbl or 230400.gbl
Which is the best way to flash the firmware?
This one:
[https://darkxst.github.io/silabs-firmware-builder/](url)
What else do I need to consider beforehand? (e.g. backup...)
Must i repaired my devices

Thanks for helping!

**Comment by Koenkk** [comment]
The recommended firmware for each adapter can be found here: https://www.zigbee2mqtt.io/guide/adapters/emberznet.html, besides the firmware, no other steps have to be taken.

**Reply by GWsmart** [reply]
@Koenkk 
Hallo and thanks for the answer!
I changed the firmware and everything has worked fine so far. Very Great!
This shows the Z2M Addon now:
Coordinator-Typ: EmberZNet
Koordinator-Version: 7.4.1 [GA]
Zigbee Herdsman Konverter Version: 19.47.1
Zigbee Herdsman Version: 0.49.2

Thank you for the work you do for the project, regards GWsmart
